app.py

```diff
@@ -88,12 +88,18 @@ def utility_processor():
     can be used in all templates
     """
 
+    if 'openid' in flask.session:
+        user_name = flask.session['openid']['fullname']
+    else:
+        user_name = None
+
```

Comment:
@tbille Is this piece OK to get user name to all pages?

Reply:
i'd rather do something like this: 
```python
if authentication.is_authenticated(flask.session):
        user_name = flask.session['openid']['fullname']
    else:
        user_name = None
```

I might add in the future a module to handle our session variables, but for now let's use this